docs/tutorials/01_introduction_cloud_runtime.ipynb

```diff
@@ -0,0 +1,734 @@
+{
```

Comment:
<blockquote>Currently the only available option is&nbsp;<code>backend_name</code>, which is optional for cloud runtime.</blockquote><p> </p><p>Which backend is chosen when the optional parameter is not provided? Why is it optional for cloud and required for legacy?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<blockquote>Which backend is chosen when the optional parameter is not provided</blockquote><p> </p><p>The one with the fewest pending jobs. It's mentioned later in the tutorial, when we show an example of running a job.</p><p> </p><blockquote>Why is it optional for cloud and required for legacy</blockquote><p> </p><p>It is much harder to do for legacy because 1) legacy has many heterogeneous backends 2) the existing hub/group/project model means the "least pending" is not necessarily the fastest. So we'll start with cloud to see how that turns out. </p>